latex-syntax/src/main/java/org/xwiki/contrib/latex/internal/TemplateProcessor.java

```diff
@@ -43,24 +48,32 @@
 
     private static final String BLOCK = "block";
 
-    private TemplateManager templateManager;
+    private final TemplateManager templateManager;
 
-    private VelocityMacroFilter filter;
+    private final VelocityMacroFilter filter;
 
-    private Map<String, Object> latexBinding;
+    private final Map<String, Object> latexBinding;
+
+    private final UIExtensionSupplier uiExtensionSupplier;
+
+    private final BlockRenderer blockRenderer;
 
     /**
      * @param templateManager the template manager used to locate, get and execute template content
-     * @param latexBinding the script context "latex" binding into which we can inject new "bindings" for the template
-     *        evaluation
+     * @param latexBinding the script context "latex" binding into which we can inject new "bindings" for the
+     *     template evaluation
```

Comment:
Same here, could be good to decide if we indent just by 4 chars or till the first letter of the parameter name (i.e. `latexBinding` in this case). I usually align on the parameter name but I don't remember if we discussed this (or if we need to ;)).

Reply:
I've left the auto-indent of my IDE decide :) I'll check if we have a rule for this and make a proposal if needed.